pkg/abstractions/image/build.go

```diff
@@ -493,6 +494,65 @@ func getPythonInstallCommand(pythonVersion string) string {
 	return fmt.Sprintf("%s && add-apt-repository ppa:deadsnakes/ppa && apt-get update && apt-get install -q -y %s && %s", baseCmd, installCmd, postInstallCmd)
 }
 
+func getPythonStandaloneInstallCommand(pythonVersion string) string {
+	var osArch string
+	switch runtime.GOARCH {
+	case "amd64":
+		osArch = "x86_64"
+	case "arm64":
+		osArch = "aarch64"
+	default:
+		log.Println("Unsupported architecture")
+	}
+
+	var venderAndOs string
+	switch runtime.GOOS {
+	case "linux":
+		venderAndOs = "unknown-linux"
+	case "darwin":
+		venderAndOs = "apple-darwin"
+	default:
+		log.Println("Unsupported OS")
+	}
+
+	// TODO: Make this configurable?
+	var pySemVer string
+	switch pythonVersion[6:] {
+	case "3.9":
+		pySemVer = "3.9.20"
```

Comment:
Where are these values from?

Reply:
Updated the code to get these from the config.